UnitedIncome/components
# PrinterIcon and PrinterCircleIcon

### Description
> Please provide a brief description of your changes below.

Adds the `PrinterIcon` and the `PrinterCircleIcon`.

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

* It's an icon, just look at it!

### Comments
> Any additional comments that will aid with the review process.

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-*)
